tests/Resources/ODS.robot

```diff
@@ -68,3 +71,36 @@ Usage Data Collection Should Not Be Enabled
     [Arguments]    ${msg}="Usage Data Collection" should not be enabled
     ${enabled}=    ODS.Is Usage Data Collection Enabled
     Should Not Be True    ${enabled}    msg=${msg}
+
+Uninstall RHODS From OSD Cluster
+    [Documentation]    Selects the cluster type and triggers the RHODS uninstallation
+    ${addon_installed}=     Is Rhods Addon Installed    ${CLUSTER_NAME}
+    Uninstall RHODS Using OLM
+    # IF    ${addon_installed} == ${TRUE}
+    #     Uninstall Rhods Using Addon    ${CLUSTER_NAME}
+    # ELSE
+    #     Uninstall RHODS Using OLM
+    # END
+
+Uninstall RHODS Using OLM
```

Comment:
[Documentation] section missing.  I guess it's uninstalling from the cluster we are logged in. How does it work? With `oc login` or `OC Login` (or both)

Reply:
it uses openshift library